lib/configuration.ts

```diff
@@ -1,30 +1,31 @@
-const path = require('path')
-const findUp = require('find-up')
-const { readFileSync } = require('fs')
+import path from 'path'
+import findUp from 'find-up'
+import { readFileSync } from 'fs'
+import { CatVConfig } from './opts'
 
 const CONFIGURATION_FILES = [
   '.versionrc',
-  '.versionrc.cjs',
   '.versionrc.json',
+  '.versionrc.cjs',
   '.versionrc.js'
-]
+] as const;
 
-module.exports.getConfiguration = function () {
-  let config = {}
+export async function getConfiguration () {
+  let config = {} as Partial<CatVConfig>
```

Comment:
Unless I'm missing something, this can be `let config: Partial<CatVConfig> = {};`

Reply:
You are correct, nice catch! I think I originally typecast ot as `CatVConfig` and added the partial after the fact. It didn't occur to me to move the type assertion to an inline type. I'll change it in my next commit.